Serialization/SerializeToJson.java

```diff
@@ -0,0 +1,63 @@
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SerializeToJson {
+    private static StringBuilder sb = new StringBuilder();
```

Comment:
Why static?

Reply:
to use SerializeToJson.serialize(group), instead of creating a SerializeToJson object